.github/workflows/cli-release.yml

```diff
@@ -0,0 +1,64 @@
+name: Release Rill CLI
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "0 0 * * *"
```

Comment:
What's the point of having this now? Shouldn't it be for nightlies only (which aren't set up yet)

Reply:
@himadrisingh Do we need this?